source/features/better-diff-selection.tsx

```diff
@@ -0,0 +1,34 @@
+import './better-diff-selection.css';
+
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+let selectedDiffTableBody: HTMLElement | undefined;
+
+function init(): void {
+	delegate(document.body, '.diff-table tr:not(.js-expandable-line) td:nth-child(even)', 'mousedown', event => {
+		const target = event.target as HTMLElement;
+		selectedDiffTableBody = target.closest('tbody')!;
+		selectedDiffTableBody.dataset.rghSelect = target.closest('td') === target.closest('tr')!.children[1] ? 'left' : 'right';
+	});
+
+	document.body.addEventListener('mouseup', () => {
+		if (selectedDiffTableBody) {
+			selectedDiffTableBody.removeAttribute('data-rgh-select');
```

Comment:
Instead of saving this `selectedDiffTableBody`, you can just use:


```js
select('[data-rgh-select]')?.removeAttribute('data-rgh-select');
```

The question mark will make sure it doesn't error if not found